spec/factories/targets.rb

```diff
@@ -6,5 +6,6 @@
     target_count 100
     unit Target.units[:submissions]
     conference
+    campaign
   end
```

Comment:
Creating a target does not require you to have a campaign, so I don't think we should have that in the factory.